schemas/user.yml

```diff
@@ -18,6 +18,7 @@
       - host
       - server
       - source
+      - package
```

Comment:
Yes

Reply:
I think this could be misleading - it could also be the user who "owns" a package or who it was installed for (e.g. for applications on Windows or macOS). I suspect in many cases we would not have the information who installed a package (i.e. Auditbeat today does not have it, and could not get it from dpkg/rpm/homebrew).

There is no great harm in putting it under `expected`, but I would not "expect" it in most cases.

Reply:
Not sure the APM agents can collect the information, but in my opinion the information about the user installing a package could be interesting for SIEM, that's why I added it. 

Would it make sense to add it as `package.installed_by`? Or would you rather not define information we are not sure our shippers can collect yet?

Reply:
Yeah, I'm hesitant to define fields that we don't fill at the moment. We can always do it later when we have more information and concrete examples.